### PR TITLE
Solved the royality fix issue.

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
This issue arised from the chain upgrade.

### CODE REVIEW

File: lib/easel_provider.dart [hunk 1]
```diff
@@ -609,7 +609,10 @@ class EaselProvider extends ChangeNotifier {
 
     disposePlayers(assetType: nft.assetType);
 
-    String residual = nft.tradePercentage.trim();
+
+
+    String tradePercentage = BigInt.from(int.parse(nft.tradePercentage.trim()) * kRoyalityPrecision).toString();
```
Comment (mustafapylons) [inline lib/easel_provider.dart]: "kRoyaltyPrecision" is a better spelling.

File: lib/utils/constants.dart [hunk 2]
```diff
@@ -92,6 +92,7 @@ const int kSixtySeconds = 60;
 const int kFileCompressQuality = 50;
 
 const double kPrecision = 100000000000000000;
+const double kRoyalityPrecision = 10000000000000000;
```
Comment (mustafapylons) [inline lib/utils/constants.dart]: "kRoyaltyPrecision" is a better spelling.

**Review by mustafapylons — COMMENTED** [review]
Some typos, otherwise code is good.

File: lib/easel_provider.dart (hunk 1 above)
Comment (kjawadDeveloper2) [reply lib/easel_provider.dart]: Solved.

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Review by Amean — APPROVED** [review]

**Pull request merged**